src/NuGet.Services.AzureSearch/SearchService/IIndexOperationBuilder.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.AzureSearch.SearchService
+{
+    public interface IIndexOperationBuilder
```

Comment:
>IIndexOperationBuilder [](start = 21, length = 22)

Consider adding a comment here (this type name is rather abstract if you don't know Azure Search)

Reply:
Done.